tensorflow/python/data/experimental/ops/autotune_options.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""API for controlling performance auto-tuning in `tf.data` pipelines."""
```

Comment:
replace "auto-tuning" / "auto-tune" with "autotuning" / "autotune" throughout this PR.

Reply:
addressed in https://github.com/tensorflow/tensorflow/pull/50291